homework.js

```diff
@@ -0,0 +1,112 @@
+//--------- 1
+const summation = function(num){
+    let result = 0;
+    for (let i = 0; i <=num ; i++){
+        result += i;
+    }
+    return result;
+}
+//----------- 2
+const evenSummation = function(num){
+    let result = 0;
+    for (let i = 0; i <=num ; i+=2){
+        result += i;
+    }
+    return result;
+}
+evenSummation(5);
+
+//------------- 3
+
+let sum = 0;
+const list = [];
+const getAvg = function(list){
+    for (let i = 0; i < list.length; i++){
+        sum += list[i];
+    }
+    let avg = sum / list.length - 1;
+    return Math.floor(avg);
+};
+//-------------- 4
+let rWord = "";
+const reverse = function(word){
+    let a = "";
+    for (let i = word.length-1 ; i >= 0 ; i--){
+         a = word.charAt(i);
+        rWord += a;
+    };
+    console.log(rWord);
+};

```

Comment:
this will do the same trick, `return word.split('').reverse().join('');`
- `split("")` will convert the string to array 
- `reverse()` will reverse the array
- `join("")` will join the array to a string